extension/source/background/AggregatorController.ts

```diff
@@ -124,6 +124,9 @@ export default class AggregatorController {
       const aggregator = new Aggregator(knownTx.aggregatorUrl);
 
       const bundleReceipt = await aggregator.lookupReceipt(hash);
+      if (bundleReceipt && 'submitError' in bundleReceipt) {
+        throw new Error(bundleReceipt.submitError);
+      }
```

Comment:
@kautukkundan @blakecduncan Is this the correct behavior (throw error in RPC handler) if there is an error with the aggregator trying to submit a bundle?

Reply:
Kautuk and I haven't caught up on this.  I think that is a good behaviour.  In the future, it may be good for us to update the transaction in the transactions table if there is a submit error.  I'd say that's outside the scope of this bug fix though because I don't think we have a good way to display a submit error yet.